lib/ImageProcessor.js

```diff
@@ -30,6 +30,7 @@ class ImageProcessor {
     resize(params) {
         return new Promise((resolve, reject) => {
             gm(this.image.getData())
+                .resize(params.width, params.height, params.resizeOption)
```

Comment:
First of all, sorry, I had a stray duplicate resize call in from an incorrect merge resolution I did, which I fixed in https://github.com/urbanriskmap/image-lambda/commit/30421e3a3d60dbbbd16c07491ae1c7e82112b993.

This remaining single call to resize already works with params.resizeOption as optional. If it's not specified in the config, then here params.resizeOption will be null, and then the call to gm(image).resize(width, height, null) acts the same as resize(width, height) - something I tested just to be sure.